test/powershell/Modules/Microsoft.PowerShell.Utility/Add-Type.Tests.ps1

```diff
@@ -1,6 +1,10 @@
 $guid = [Guid]::NewGuid().ToString().Replace("-","")
 
 Describe "Add-Type" -Tags "CI" {
+    It "Public 'Language' enumeration contains all members" {
+        [Enum]::GetNames("Microsoft.PowerShell.Commands.Language") -join "," | Should Be "CSharp,CSharpVersion7,CSharpVersion6,CSharpVersion5,CSharpVersion4,CSharpVersion3,CSharpVersion2,CSharpVersion1,VisualBasic,JScript"
+    }
+
```

Comment:
We checked it manually in the original Issue.
I didn't touch the functionality of this cmdlet. Besides, we don't even have tests for it right now. So I believe this is a separate huge job.
It seems @daxian-dbw is working on Add-Type. It would be good to agree on the distribution of work on tests.

I can say nothing about separate commits. 😕